packages/react-components/react-tabster/src/hooks/useArrowNavigationGroup.ts

```diff
@@ -43,8 +56,9 @@ export const useArrowNavigationGroup = (options: UseArrowNavigationGroupOptions
     mover: {
       cyclic: !!circular,
       direction: axisToMoverDirection(axis ?? 'vertical'),
-      memorizeCurrent: memorizeCurrent,
-      tabbable: tabbable,
+      memorizeCurrent,
+      tabbable,
+      hasDefault: !!unstable_hasDefault,
```

Comment:
let's make hasDefault undefined, to avoid changing snapshots both in Fluent and for partners